app/scripts/controllers/create.js

```diff
@@ -52,27 +52,43 @@ angular.module('openshiftConsole')
 
         // List image streams and templates in the both the shared `openshift`
         // namespace and the project namespace.
-        DataService.list("imagestreams", {namespace: "openshift"}, function(imageStreams) {
-          $scope.openshiftImageStreams = imageStreams.by("metadata.name");
-        });
+        DataService
+          .list("imagestreams", {namespace: "openshift"}, null, {
+            errorNotification: true
+          })
+          .then(function(resp) {
+            $scope.openshiftImageStreams = resp.by("metadata.name");
+          });
```

Comment:
```js
Data.list("imagestreams", {namespace: "openshift"}).then(function(resp) {
  $scope.openshiftImageStreams = resp.by("metadata.name");
});
```

Leave the extra arguments off this request and all of the others below to take the defaults.